src/settingsClient.js

```diff
@@ -0,0 +1,305 @@
+/* eslint-disable */
+import axios from 'axios'
+import qs from 'qs'
+let domain = ''
+export const getDomain = () => {
+  return domain
+}
+export const setDomain = ($domain) => {
+  domain = $domain
+}
+export const request = (method, url, body, queryParameters, form, config) => {
```

Comment:
I guess one can pass in headers in "config" ?

Reply:
FYI, bearer token is injected automatically here: https://github.com/owncloud/owncloud-sdk/blob/440966fc7071ea27ee91195bfcf3202e6f7d4851/src/helperFunctions.js#L61